scripts/estimate-builds.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# pylint: disable=invalid-name
+
+import datetime
+from pathlib import Path
+import sys
+
+import croniter
+
+# Add the root of the repo to PYTHONPATH for utils
+ci_root = Path(__file__).resolve().parents[1]
+sys.path.insert(0, str(ci_root))
+# pylint: disable-next=wrong-import-position
+from utils import get_config_from_generator  # noqa: E402
+
+config = get_config_from_generator()
+
+builds_per_tree = {}
+for tree in config['tree_schedules']:
+    tree_name = tree['name']
+    tree_llvm_ver = tree['llvm_version']
```

Comment:
Sink these closer to their first use, or get rid of them; this loop uses a mix of local vars and dict access inconsistently.

Reply:
Sure: e89a021ac9d4b4f01ed7f747f66c43109606db15

I opted for variables when I need the value from the dictionary in more than one place.